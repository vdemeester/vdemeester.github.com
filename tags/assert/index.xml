<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>assert on Vincent Demeester</title>
    <link>https://vincent.demeester.fr/tags/assert/</link>
    <description>Recent content in assert on Vincent Demeester</description>
    <generator>Hugo -- gohugo.io</generator>
    <lastBuildDate>Thu, 16 Aug 2018 00:00:00 +0200</lastBuildDate>
    
	<atom:link href="https://vincent.demeester.fr/tags/assert/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Golang testing — gotest.tools assertions</title>
      <link>https://vincent.demeester.fr/posts/2018-08-16-gotest-tools-assertions/</link>
      <pubDate>Thu, 16 Aug 2018 00:00:00 +0200</pubDate>
      
      <guid>https://vincent.demeester.fr/posts/2018-08-16-gotest-tools-assertions/</guid>
      <description>Table of Contents  Assert and Check More assert helpers cmp.Comparison  Equality with Equal and DeepEqual Errors with Error, ErrorContains and ErrorType Bonus with Panics Miscellaneous with Contains, Len and Nil Write your own Comparison  Conclusion…   
Assert and Check Both those functions accept a Comparison (we&amp;rsquo;ll check what it is later on) and fail the test when that comparison fails. The one difference is that Assert will end the test execution at immediately whereas Check will fail the test and proceed with the rest of the test case.</description>
    </item>
    
  </channel>
</rss>