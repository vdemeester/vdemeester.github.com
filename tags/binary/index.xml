<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Binary on Vincent Demeester</title>
    <link>https://vincent.demeester.fr/tags/binary/</link>
    <description>Recent content in Binary on Vincent Demeester</description>
    <generator>Hugo -- gohugo.io</generator>
    <lastBuildDate>Tue, 23 Apr 2019 00:00:00 +0000</lastBuildDate>
    
	<atom:link href="https://vincent.demeester.fr/tags/binary/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Trimming the fat from a Golang binary</title>
      <link>https://vincent.demeester.fr/posts/2019-04-23-link/</link>
      <pubDate>Tue, 23 Apr 2019 00:00:00 +0000</pubDate>
      
      <guid>https://vincent.demeester.fr/posts/2019-04-23-link/</guid>
      <description></description>
    </item>
    
  </channel>
</rss>