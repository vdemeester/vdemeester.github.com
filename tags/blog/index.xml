<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Blog on Vincent Demeester</title>
    <link>https://vincent.demeester.fr/tags/blog/</link>
    <description>Recent content in Blog on Vincent Demeester</description>
    <generator>Hugo -- gohugo.io</generator>
    <lastBuildDate>Sat, 09 May 2015 00:00:00 +0000</lastBuildDate>
    
	<atom:link href="https://vincent.demeester.fr/tags/blog/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Migration vers hugo</title>
      <link>https://vincent.demeester.fr/posts/2015-05-09-migration-to-hugo/</link>
      <pubDate>Sat, 09 May 2015 00:00:00 +0000</pubDate>
      
      <guid>https://vincent.demeester.fr/posts/2015-05-09-migration-to-hugo/</guid>
      <description>Et voilà, une semaine après m&#39;être remis en route sur ce blog, je fais déjà une migration &amp;gt;_&amp;lt;. Je me suis rendu compte que Jekyll m&#39;énervait un peu, et comme je suis à fond dans go en ce moment, j&#39;ai regardé du côté des générateurs de sites static et Hugo a clairement retenu mon attention. Pourquoi Hugo ?   Make the Web Fun Again  Introducing Hugo, a new idea around making website creation simple again.</description>
    </item>
    
    <item>
      <title>Orgmode et Jekyll</title>
      <link>https://vincent.demeester.fr/posts/2015-05-01-orgmode-et-jekyll/</link>
      <pubDate>Fri, 01 May 2015 00:00:00 +0000</pubDate>
      
      <guid>https://vincent.demeester.fr/posts/2015-05-01-orgmode-et-jekyll/</guid>
      <description>Un peu plus d&#39;un an..  .. que je n&#39;ai rien posté ici. Le temps passe bien vite, mais c&#39;est plutôt une bonne chose. Je crois que quelque chose me manquait pour poster ici plus souvent.   Au niveau de mes points d&#39;entrées sur le web, je compte remettre un peu en route la partie blog de ce site et tourner shortbrain.org (ou autre) en un site plus &#34;documentation&#34; en me basant sur mes notes (powered by org-mode).</description>
    </item>
    
  </channel>
</rss>