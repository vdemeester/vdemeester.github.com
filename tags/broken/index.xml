<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Broken on Vincent Demeester</title>
    <link>https://vincent.demeester.fr/tags/broken/</link>
    <description>Recent content in Broken on Vincent Demeester</description>
    <generator>Hugo -- gohugo.io</generator>
    <lastBuildDate>Fri, 19 Apr 2019 00:00:00 +0000</lastBuildDate>
    
	<atom:link href="https://vincent.demeester.fr/tags/broken/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Stop writing broken Go libraries</title>
      <link>https://vincent.demeester.fr/posts/2019-04-19-link/</link>
      <pubDate>Fri, 19 Apr 2019 00:00:00 +0000</pubDate>
      
      <guid>https://vincent.demeester.fr/posts/2019-04-19-link/</guid>
      <description></description>
    </item>
    
  </channel>
</rss>