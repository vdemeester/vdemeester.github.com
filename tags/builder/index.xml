<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Builder on Vincent Demeester</title>
    <link>http://vincent.demeester.fr/tags/builder/</link>
    <description>Recent content in Builder on Vincent Demeester</description>
    <generator>Hugo -- gohugo.io</generator>
    <lastBuildDate>Sun, 01 Jan 2017 00:00:00 +0000</lastBuildDate>
    
	<atom:link href="http://vincent.demeester.fr/tags/builder/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Golang testing â€” functional arguments for wonderful builders</title>
      <link>http://vincent.demeester.fr/posts/2017-01-01-go-testing-functionnal-builders/</link>
      <pubDate>Sun, 01 Jan 2017 00:00:00 +0000</pubDate>
      
      <guid>http://vincent.demeester.fr/posts/2017-01-01-go-testing-functionnal-builders/</guid>
      <description>Programming is not easy; even the best programmers are incapable of writing programs that work exactly as intended every time. Therefore an important part of the software development process is testing. Writing tests for our code is a good way to ensure quality and improve reliability.   Go programs, when properly implemented, are fairly simple to test programatically. The testing built-in library and the features of the language itself offer plenty of ways to write good tests.</description>
    </item>
    
  </channel>
</rss>