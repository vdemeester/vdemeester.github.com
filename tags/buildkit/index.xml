<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>buildkit on Vincent Demeester</title>
    <link>https://vincent.demeester.fr/tags/buildkit/</link>
    <description>Recent content in buildkit on Vincent Demeester</description>
    <generator>Hugo -- gohugo.io</generator>
    <lastBuildDate>Fri, 15 Feb 2019 00:00:00 +0000</lastBuildDate>
    
	<atom:link href="https://vincent.demeester.fr/tags/buildkit/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>$ docker build -f Mockerfile.yaml .</title>
      <link>https://vincent.demeester.fr/posts/2019-02-15-link/</link>
      <pubDate>Fri, 15 Feb 2019 00:00:00 +0000</pubDate>
      
      <guid>https://vincent.demeester.fr/posts/2019-02-15-link/</guid>
      <description>how to write your own Dockerfile syntax that works out of the box with any existing Docker installation.</description>
    </item>
    
  </channel>
</rss>