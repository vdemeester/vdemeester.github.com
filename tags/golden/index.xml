<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Golden on Vincent Demeester</title>
    <link>http://vincent.demeester.fr/tags/golden/</link>
    <description>Recent content in Golden on Vincent Demeester</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>fr-fr</language>
    <lastBuildDate>Sat, 22 Apr 2017 00:00:00 +0000</lastBuildDate>
    
	<atom:link href="http://vincent.demeester.fr/tags/golden/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Golang testing â€” golden file</title>
      <link>http://vincent.demeester.fr/posts/2017-04-22-golang-testing-golden-file/</link>
      <pubDate>Sat, 22 Apr 2017 00:00:00 +0000</pubDate>
      
      <guid>http://vincent.demeester.fr/posts/2017-04-22-golang-testing-golden-file/</guid>
      <description>Tests are all about maintainability and readability. You want the least visual noise possible and it should not be a hassle to maintain. When testing functions that output a long of string, in case of a command line output testing, readability and maintainance tend to be tricky to achieve. The problem  As an example let&#39;s say we want to test out the output of a command that displays a list as a table.</description>
    </item>
    
  </channel>
</rss>